### ISSUE TITLE
Unable to login using different types of wine 

### ISSUE THREAD

**Issue by Tomar753OG** [yaagl/yet-another-anime-game-launcher#371]
I was using the whisky-dvxk wine option in YAAGL and it told me to download the latest game file from Genshin's website. However once downloaded , it forces me to download the new game launcher " Hoyoplay" that mihoyo recently brought out and it appeared blank for me. I tried using other wines( 9.2,8.16) on YAAGL but it still doesn't work 

<img width="1728" alt="Screenshot 2024-06-19 at 3 31 44 AM" src="https://github.com/yaagl/yet-another-anime-game-launcher/assets/173193285/f5bfa148-67e0-49b6-bb16-4cbb13826af4">
![Screenshot 2024-06-19 at 3 32 48 AM](https://github.com/yaagl/yet-another-anime-game-launcher/assets/173193285/fbd13ba6-87e4-43c8-bda3-f2243b29f685)

 - macOS: [14.5 Sonoma]
 - Device: [Macbook pro 16 inch  ]
 - CPU: [M2 max]
 - RAM: [ 64g]

**Additional context**
9.2v and 8.16v would not even react after launch game button was clicked

**Comment by Menelkir** [comment]
I don't follow, where it says that it forces you to use hoyoplay?

**Comment by Tomar753OG** [comment]
> I don't follow, where it says that it forces you to use hoyoplay?

[
![IMG_7864](https://github.com/yaagl/yet-another-anime-game-launcher/assets/173193285/caff0c05-84e4-4ca9-b8f7-847cbd3f6d70)
](url)

**Comment by Tomar753OG** [comment]
![IMG_7866](https://github.com/yaagl/yet-another-anime-game-launcher/assets/173193285/fdf1d49a-d60c-4f4d-89fb-5efdbaa92839)

Since the old launcher wasn’t allowed anymore, I was told to download hoyoplay in whisky and it appeared to be blank

**Comment by Zqanx** [comment]
Toggle ac patch in yaagl. Why are you using whisky?

**Comment by Menelkir** [comment]
> > I don't follow, where it says that it forces you to use hoyoplay?
> 
> [ ![IMG_7864](https://private-user-images.githubusercontent.com/173193285/340816965-caff0c05-84e4-4ca9-b8f7-847cbd3f6d70.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NTMyMTIsIm5iZiI6MTcxODc1MjkxMiwicGF0aCI6Ii8xNzMxOTMyODUvMzQwODE2OTY1LWNhZmYwYzA1LTg0ZTQtNGNhOS1iOGY3LTg0N2NiZDNmNmQ3MC5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MThUMjMyMTUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njk5ZDQ2OWNmNmE0MmE3MDY1YmJkMzQ4OTQ0Njc3MDY1YTBiOThkMTY2MTYzZTdlN2U3NDIwZmIwNzUwY2I0ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.cr3ee5pyErtH7BPxF2trk_LOUDeY_XnWdbLlQaSiULY) ](url)

Are you using the official launcher under whisky? And how this is related to yaagl?

**Comment by Sipperz** [comment]
You are supposed to go in Yaagl settings > Wine > change the current wine to "whisky"

NOT install the game in a whisky bottle.

**Comment by Tomar753OG** [comment]
what if it still shows data error ? should i uninstall yaagl and download it again ? 
> You are supposed to go in Yaagl settings > Wine > change the current wine to "whisky"
> 
> NOT install the game in a whisky bottle.
> 
> Yaagl is a different launcher, Whisky is emulating the original exe.

**Comment by Sipperz** [comment]
> what if it still shows data error ? should i uninstall yaagl and download it again ?
> 
> > You are supposed to go in Yaagl settings > Wine > change the current wine to "whisky"
> > NOT install the game in a whisky bottle.
> > Yaagl is a different launcher, Whisky is emulating the original exe.

show me the Error first

**Comment by Tomar753OG** [comment]
> show me the Error first

<img width="1728" alt="Screenshot 2024-06-19 at 9 33 54 PM" src="https://github.com/yaagl/yet-another-anime-game-launcher/assets/173193285/1c2add1d-f425-4611-890a-24ce6bb78799">

<img width="1728" alt="Screenshot 2024-06-19 at 3 31 44 AM" src="https://github.com/yaagl/yet-another-anime-game-launcher/assets/173193285/9ffb8c2d-9a7c-4abf-8c23-454c4a7ba78b">

**Comment by Sipperz** [comment]
Toggle AC patch in settings

**Comment by Tomar753OG** [comment]
> Toggle AC patch in settings

i've tried it before, doesn't work. Same system notification showing data error still pops out

**Comment by Sipperz** [comment]
where are Your game files and Yaagl located?

Make sure Yaagl is in /Applications and Game files are all in one folder where Icloud is not accessing stuff.

Since u have 64GB RAM might also consider using Crossover in Wine option than Whisky it performs way better.

**Comment by Tomar753OG** [comment]
ive got games files in my user folder and yaagl in applications. it worked well for me for quite a few days and was playing casually until yesterday when i login it showed me data error 

> where are Your game files and Yaagl located?
> 
> Make sure Yaagl is in /Applications and Game files are all in one folder where Icloud is not accessing stuff.
> 
> Since u have 64GB RAM might also consider using Crossover in Wine option than Whisky it performs way better.

**Comment by Sipperz** [comment]
I see, have u tried downloading the "pre-release" version from Github? Maybe that will solve the issue

**Comment by Tomar753OG** [comment]
> I see, have u tried downloading the "pre-release" version from Github? Maybe that will solve the issue



yeah i did, turns out its still not working. imma uninstall everything and set up yaagl from beginning. not planing to switch over to crossover tho cuz my trial went expired

**Comment by Zqanx** [comment]
Yaagl uses crossover as a wine distribution. Crossover's trial doesn't affect yaagl using crossover's wine. We're not telling you to run GI directly under crossover.

**Comment by Sipperz** [comment]
Did it work with the full reinstall? Will keep the issue open for 1 week then close it if no response, feel free to re-open the issue after if required.

**Comment by ScathachSkadi** [comment]
Closing because no response and use DXMT now.
